src/platform/user/profile/vet360/components/base/TransactionErrorBanner.jsx

```diff
@@ -12,16 +12,11 @@ export function GenericUpdateError(props) {
   return (
     <AlertBox
       {...props}
+      className="vads-u-margin-bottom--4"
       status="error"
-      content={
-        <div>
-          <h4>Your recent profile update didn’t save</h4>
-          <p>
-            We’re sorry. Something went wrong on our end and we couldn’t save
-            the recent updates you made to your profile. Please try again later.
-          </p>
-        </div>
-      }
+      headline="Your recent profile update didn’t save"
+      content="We’re sorry. Something went wrong on our end and we couldn’t save the
+      recent updates you made to your profile. Please try again later."
```

Comment:
I'm fine with this being included in this PR either way, but does this change have any impact on the error described in the PR description?

Reply:
no, but it fixes the issue with the error alert looking bad. this screenshot shows the "before", including the left padding in the Hero mentioned above.

![image](https://user-images.githubusercontent.com/20728956/66341691-ab9b6000-e8fc-11e9-8266-17efcbeeae3c.png)

Reply:
@mojaray2k can you duplicate this fix for the other AlertBox-rending components in this file?

Reply:
@erikphansen Done